app/assets/stylesheets/components/_tax-return-assignees.scss

```diff
@@ -1,5 +1,47 @@
 .tax-return-assignees {
+  margin: 0;
   li {
+    display: flex;
+    line-height: 3.5rem;
+    align-items: center;
     margin-bottom: 5px;
+    > div {
+      margin-right: 5px;
+    }
+  }
+
+  .assignment {
+    display: flex;
+    flex-grow: 1;
```

Comment:
the LI is flexed, and there are two children: the assignment side and the status side. Setting this to flex-grow: 1 should indicate that the assignment side should fill the extra available space for the section because it is greater than the default on the other side which is flex-grow: 0